profiles/Tier30/T30_Warrior_Arms.simc

```diff
@@ -108,24 +108,27 @@ actions.hac+=/wrecking_throw
 actions.single_target=sweeping_strikes,if=spell_targets.whirlwind>1
 actions.single_target+=/mortal_strike
 actions.single_target+=/rend,if=remains<=gcd|talent.tide_of_blood&cooldown.skullsplitter.remains<=gcd&(cooldown.colossus_smash.remains<=gcd|debuff.colossus_smash.up)&dot.rend.remains<dot.rend.duration*0.85
-actions.single_target+=/avatar,if=talent.warlords_torment&rage.pct<33&(cooldown.colossus_smash.ready|debuff.colossus_smash.up|buff.test_of_might.up)|!talent.warlords_torment&(cooldown.colossus_smash.ready|debuff.colossus_smash.up)
+actions.single_target+=/avatar,if=cooldown.colossus_smash.ready|debuff.colossus_smash.up|buff.test_of_might.up
 actions.single_target+=/spear_of_bastion,if=cooldown.colossus_smash.remains<=gcd|cooldown.warbreaker.remains<=gcd
 actions.single_target+=/warbreaker
 actions.single_target+=/colossus_smash
 actions.single_target+=/thunderous_roar,if=buff.test_of_might.up|talent.test_of_might&debuff.colossus_smash.up&rage.pct<33|!talent.test_of_might&debuff.colossus_smash.up
 actions.single_target+=/bladestorm,if=talent.hurricane&(buff.test_of_might.up|!talent.test_of_might&debuff.colossus_smash.up)|talent.unhinged&(buff.test_of_might.up|!talent.test_of_might&debuff.colossus_smash.up)
 actions.single_target+=/skullsplitter,if=talent.tide_of_blood&dot.rend.remains&(debuff.colossus_smash.up|cooldown.colossus_smash.remains>gcd*4&buff.test_of_might.up|!talent.test_of_might&cooldown.colossus_smash.remains>gcd*4)|rage<30
-actions.single_target+=/execute,if=buff.sudden_death.react
+actions.single_target+=/execute,if=buff.sudden_death.react&debuff.colossus_smash.down
 actions.single_target+=/shockwave,if=talent.sonic_boom.enabled
 actions.single_target+=/ignore_pain,if=talent.anger_management|talent.test_of_might&debuff.colossus_smash.up
-actions.single_target+=/whirlwind,if=talent.storm_of_swords&talent.battlelord&rage.pct>80&debuff.colossus_smash.up
+actions.single_target+=/whirlwind,if=talent.storm_of_swords&talent.battlelord&debuff.colossus_smash.up
```

Comment:
The extra WW attempt also results frequently in less overpowered casts. Overpowered is usually a DPS loss during the colossus smash window.